### DISCUSSION TITLE
Johannes Schickling

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#284] [Announcements]
- https://twitter.com/schickling?lang=en
- http://effect.website
- https://overtone.pro/
- https://www.localfirst.fm/
- https://www.prisma.io/

### Prisma

- [x] What problems led you to founding prisma? From what it looks like an open source tool grew into that company.

### Effect 

- [x] What is effect and how does it change how I write code?
- [x] For it to be effective do I have to write all my code in this new style?
- [x] The learning barrier here seems quite high. How would you suggest approaching it if you've never programmed like this before?
- [x] At some point in the docs everything switches to generators? How do those change the code I write?
- [ ] ~Effect seems packed with powerful features. What are some of your favorite things it makes easy?~

### Local First

- [x] What aspects of local first dev get you excited? Also name some cool local first apps.
- [x] You're building a local first app, what is it? How has local first made it better?
- [x] What are some challenges you've had building it?
- [x] What are some packages/libraries you've created to enable builders in the space?
- [x] What do you think the best local first stack is right now?
- [x] You've typically been far away form the front end but for your music player you've been using react. How have you liked it and what are some things you've learned?

### Future

- [x] You help other devtools companies. How do you decide who to work with?
- [x] What do you think the future of development is like? is it all local first? Do we all own our data?

**Comment by hipstersmoothie** [comment]
https://github.com/rossjrw/pr-preview-action
https://typedoc.org/
https://github.com/PetoiCamp/OpenCat

**Comment by schickling** [comment]
- [Nix](https://nixos.org)
- [Attest](https://github.com/arktypeio/arktype/tree/2.0/ark/attest#cli) + https://ui.perfetto.dev/

**Comment by just-be-dev** [comment]
- https://unstorage.unjs.io/
- https://github.com/OnedocLabs/react-print-pdf
